docs/layouts.md

```diff
@@ -0,0 +1,130 @@
+---
+title: Blazor layouts
```

Comment:
As a general point, I think it would be good if we didn't repeat the word "Blazor" so much. Since this is the Blazor documentation, that context is already established. Could we just say "Layouts" everywhere in this doc rather than branding them as "Blazor layouts"? Thanks 😄

Reply:
@rstropek ... "Blazor" should stay in the title and the H1:

* All of the doc titles must say "Blazor" somewhere for SEO.
* The H1 should mirror the title. It's not a MS style manual requirement, but that's what we do in the aspnet/Docs repo. We're carrying that over in this doc set.
* Having said that tho, the title doesn't need to read *Blazor layouts*. Other options include: *Layouts in Blazor*, *Use layouts to avoid markup and code duplication in Blazor*, etc. ... so long as "Blazor" is in there somewhere. `@danroth27` may have a preference tho. This one, *Blazor layouts*, was approved in our high-level planning.

Regarding the rest of the references, yes, sounds good ... "layouts" unless you need to distinguish from other non-Blazor layouts.

Reply:
Ok. Will leave Blazor in the title and remove it in the rest of the text where appropriately.